.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "eslint.enable": true
+}
```

Comment:
Please remove, we shouldn't check in editor specific files.

Reply:
yup, totally just a mistake. Thanks for catching!